language-support/qute/qute.tmLanguage.json

```diff
@@ -47,9 +47,43 @@
       ]
     },
     "parameter_declaration": {
-      "begin": "(?<!\\\\){@",
-      "end": "(?<!\\\\)}",
-      "name": "keyword.other.qute"
+      "begin": "({)(@)",
+      "beginCaptures": {
+				"1": {
+					"name": "keyword.other.qute"
```

Comment:
I would use the same name than expression or section in other words 
support.constant.handlebars" but I would rename all handlebars with qute because it is not syntax coloration for handlebars but for qute.